source/fuzz/transformation_add_synonym.cpp

```diff
@@ -211,7 +224,7 @@ TransformationAddSynonym::MakeSynonymousInstruction(
           (synonym_type->AsVector() &&
            synonym_type->AsVector()->element_type()->AsInteger()) ||
           synonym_type->AsInteger();
-      auto opcode = SpvOpNop;
+
```

Comment:
You can move the `is_integral` variable out of the `switch` as well. This will allow you to flatten the switch below.

Reply:
Done